packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -308,6 +331,77 @@ class RenderUiKitView extends RenderBox {
 
   @override
   bool hitTestSelf(Offset position) => hitTestBehavior != PlatformViewHitTestBehavior.transparent;
+
+  @override
+  void handleEvent(PointerEvent event, HitTestEntry entry) {
+    if (event is PointerDownEvent) {
+      _gestureRecognizer.addPointer(event);
+    }
+  }
+
+  @override
+  void detach() {
+    _gestureRecognizer.reset();
+    super.detach();
+  }
+}
+
+class _UiKitViewGestureRecognizer extends OneSequenceGestureRecognizer {
+  _UiKitViewGestureRecognizer(this.controller, this.gestureRecognizerFactories) {
+    team = GestureArenaTeam();
+    team.captain = this;
+    _gestureRecognizers = gestureRecognizerFactories.map(
+            (Factory<OneSequenceGestureRecognizer> factory) {
+          return factory.constructor()..team = team;
+        }
+    ).toSet();
+  }
+
+
+  // We use OneSequenceGestureRecognizers as they support gesture arena teams.
+  // TODO(amirh): get a list of GestureRecognizers here.
+  // https://github.com/flutter/flutter/issues/20953
+  final Set<Factory<OneSequenceGestureRecognizer>> gestureRecognizerFactories;
+  Set<OneSequenceGestureRecognizer> _gestureRecognizers;
+
+  final UiKitViewController controller;
+
+  @override
+  void addPointer(PointerDownEvent event) {
+    startTrackingPointer(event.pointer);
+    for (OneSequenceGestureRecognizer recognizer in _gestureRecognizers) {
+      recognizer.addPointer(event);
+    }
+  }
+
+  @override
+  String get debugDescription => 'UIKit view';
+
```

Comment:
The doc comment for this field says "**very short** pretty description", I guess that is because it is used for concatenating all the the recognizers of a `RawGestureDetector` into a single descriptive string. So I'd keep it short.
Anyway right now this code path is not used as this is private and is not used with a raw gesture detector, it's mainly here to implement the full interface.